ffi/ecma402/Cargo.toml

```diff
@@ -29,6 +29,7 @@ all-features = true
 ecma402_traits = { version = "0.2.0" }
 icu = { path = "../../components/icu", default-features = false }
 icu_provider = { version = "0.2", path = "../../provider/core" }
+icu_plurals = { version = "0.2", path = "../../components/plurals", features = ["std"] }
```

Comment:
I had to add this because ecma402_traits requires error types to implement `Error`

Reply:
Filed https://github.com/google/rust_icu/issues/207